### ISSUE TITLE
The loss function for EBM

### ISSUE THREAD

**Issue by JWKKWJ123** [interpretml/interpret#580]
Hi all,
As far as I know, the loss function for EBM is cross-entropy for classification tasks and MSE for regression tasks. However, I’m not sure what the loss function is for each shape function (decision tree) used within the model. Also, is it possible to modify the loss function for each shape function?

**Comment by paulbkoch** [comment]
Hi @JWKKWJ123 -- The shape functions, or the trees that compose them, do not have individual loss functions.  The loss function includes the target value and the additive sum of the shape functions with the intercept.  In theory, you could have a model without any features, and you would still be able to calculate the loss from just the intercept and target.

Yes, the loss function for classification is cross-entropy loss and MSE for regression.

**Comment by JWKKWJ123** [comment]
> Hi @JWKKWJ123 -- The shape functions, or the trees that compose them, do not have individual loss functions. The loss function includes the target value and the additive sum of the shape functions with the intercept. In theory, you could have a model without any features, and you would still be able to calculate the loss from just the intercept and target.
> 
> Yes, the loss function for classification is cross-entropy loss and MSE for regression.

Hi Paul,
Thank you very much! now i am clear.
